test/OpenTelemetry.ClrProfiler.Managed.Bootstrapping.Tests/InstrumentationTests.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Diagnostics;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.ClrProfiler.Managed.Bootstrapping.Tests
+{
+    public class InstrumentationTests
```

Comment:
wouldn't these tests fail if all are being run using `dotnet test`? 

if so would it not better to e.g. require some env var to run a test like `BOOSTRAPPING_TESTS` and use https://josephwoodward.co.uk/2019/01/skipping-xunit-tests-based-on-runtime-conditions to check if is set?

Reply:
I see that this pattern is used in other places `[Trait("Something", "True")]`

Reply:
I'm not really sure how that would work. I mean, yes if you just run dotnet test on the whole projects 3 out of 4 tests will fail, but I don't know how adding the env variable can resolve that.

Reply:
I can try making a PR tomorrow morning.

Reply:
Another alternative is to consider this as more of an integration test and in the test you can programmatically start a new process with the appropriate environment variables. It will likely be more complicated if you need more "bootstrapping" code to get a test like that working.

Reply:
If you are able to add a layer of indirection to the call to `OpenTracing.Util.GlobalTracer.RegisterIfAbsent` in the `Initialize` method you'd be able to swap out that call with a mocked call here and not have to worry about managing a different process or using a different test project. You just need to ensure that the original call is replaced after the test completes.

Reply:
I pushed my proposal directly to your branch.
Let me know what yoy think: 9eaac502f

Reply:
> Another alternative is to consider this as more of an integration test and in the test you can programmatically start a new process with the appropriate environment variables

I prefer the current approach as running as a separate process would also test the Profiler+Loader code. The current approach is more granural.